gxsort/sort.go

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gxsort
+
+import (
+	"sort"
+)
+
+func SortInt64(slice []int64) {
+	sort.Sort(Int64Slice(slice))
```

Comment:
why not 

```
func lessInt64(){...}

sort.Slice(slice, lessInt64)
```
```

Reply:
This implementation refers to the golang's sort-library implementation:
![image](https://user-images.githubusercontent.com/9923838/71795140-b68e9980-307f-11ea-8e5d-b235cc6b2e7a.png)